packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -16,6 +17,8 @@ import { useFloatingSuggestionItems } from './hooks/useFloatingSuggestionItems';
 import { useSelectedItems } from './hooks/useSelectedItems';
 import { IFloatingSuggestionItemProps } from '../../FloatingSuggestionsComposite';
 import { copyToClipboard } from '../SelectedItemsList/index';
+import { getTheme, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { IDragDropEvents } from 'office-ui-fabric-react';
```

Comment:
nit: could you combine this with other imports from office-ui-fabric-react on line 6